mkdocs.yml

```diff
@@ -1,12 +1,2 @@
 site_name: RTFM
-site_dir: /var/www/rtfm/ 
-#pages:
-#    - Home: index.md
-#    - Docs:
-#      - Documentation: docs.md
-#      - c220: c220.md
-#      - Flatline: flatline.md
-#      - lxd-member-boxes: lxd-member-boxes-faq.md
-#      - Motioneye: motioneye.md
-#      - Pi-Hole & Stubby: pi-hole-stubby.md
-#      - Unifi: unifi.md
+site_dir: /var/www/rtfm/
```

Comment:
@jamcole  - we need this for travis/crayon/rtfm-generated?

Reply:
Output directory isn't needed for crayon-architect since we already it with a command line argument in crayon-architect.conf.sh